### DISCUSSION TITLE
Question- Is There a Rule Against Adding Two New Topics in One Pull Request?

### DISCUSSION THREAD

**Discussion by magmaleadermaxie** [Monika-After-Story/MonikaModDev#8190] [QA - tech support + general] [answered]
I want to update my topic pull request to include another topic, the other topic only occurs after the previous topic is seen. Would this be allowed? Please forgive me if this is a dumb question, I am a newer community member.

**Comment by lunulae** [comment, marked as answer]
It's not a dumb question at all, thank you for checking!

Multiple topics in one PR is okay if they're related. We do prefer to keep them separated if they start to get really long, just for ease of reviewing, but two related ones should be fine.

Also, for future reference, if you ever write topics that need to be released at the same time (i.e. both wouldn't work without the other for some contextual reason) having them in the same PR does guarantee they'll be merged at the same time.

**Reply by magmaleadermaxie** [reply]
Thank you!
